index.html

```diff
@@ -323,5 +323,28 @@ <h3 class="heading-symbol3">
           </div>
         </div>
       </section> 
+      <section id="contact-form">
+        <div class="contact-container">
+          <div class="contact-option">
+            <article class="contact-option-lists">
+              <p>I'm always intersted in hearing about now projects, so if you wouldn't like to chat please get in</p>
+            </article>
+          </div>
+          <form action="https://formspree.io/f/mbjwvgdz" method="post">
+            <ul class="contact-form">
+              <li>
+                <input type="text" id="name" name="user_name" placeholder="Full name">
+              </li>
+              <li>
+                <input type="email" id="mail" name="user_email" placeholder="Email Adress">
+               </li>
+               <li>
+                <textarea id="msg" name="user_message" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Please add `maxlength` property and set its value to 500. And don't forget to add `required` property here as well.